models/aerodrome/base/aerodrome_base_trades.sql

```diff
@@ -7,105 +7,39 @@
     file_format = 'delta',
     incremental_strategy = 'merge',
```

Comment:
i know you're still in draft stage, but don't forget to update config blocks to be materialized as views. then we should be good to have just a few options in config block:
`materialized = view`
`schema = ...`
`alias = ...`
`post_hook ...` (optional at times)

Reply:
So I will just keep 3~4 fields in the config block:
- schema
- alias
- materialized
- post_hook (optional)